### ISSUE TITLE
Is this different enough from uralex?

### ISSUE THREAD

**Issue by xrotwang** [lexibank/syrjaenenuralic#1]
See https://github.com/lexibank/uralex

**Comment by LinguList** [comment]
Yes, it is, as it was used in List et al. (2017), as part of six by then not yet cldf datasets, which were by then modified (similar to sidwellbahnaric, starostintujia, etc.). I add these now to have them in CLDF, as they represent legacy data that was used. It does not mean the data needs to be featured anywhere specifically. Uralex also does not have phonetic transcriptions. This one does.

**Comment by LinguList** [comment]
But uralex should be added to clics3 or clics4, I just saw ;)

**Comment by xrotwang** [comment]
Uralic language data might be a perfect example for some quality / inter-annotater-agreement checking.

**Comment by LinguList** [comment]
Yes, e.g., we could check the 7 languages against Uralex (which is the same, of course), or against NorthEuralex.

**Comment by xrotwang** [comment]
And IDS has 13 Uralic languages, too.

**Comment by xrotwang** [comment]
I guess, that could be the next step after "lexibank-core": Go from just selecting gold-standard datasets to **synthesizing** "best" datasets per family.
